css/activities.css

```diff
@@ -90,6 +90,64 @@
   font-size: large;
 }
 
+.trash-view {
+  position: relative;
+  background-color: white;
+  max-width: 396px;
+  max-height: 200px;
+  overflow-y: auto;
+  font-size: 16px;
+  color: black;
+  border: 2px solid #87cefa;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  text-align: left;
+}
+
+.button-container {
```

Comment:
Maybe the background for this container should be the same blue we use for toolbars.
Maybe the buttons should be icons with hover help text instead of labels. (We'll need to decide on what icons.)